### PR TITLE
Import from dir

### PR DESCRIPTION
**Pull request by cazfletch**

### CODE REVIEW

File: packages/blockchain-common/src/registries/FabricWalletRegistry.ts [hunk 1]
```diff
@@ -51,4 +54,49 @@ export class FabricWalletRegistry extends FileRegistry<FabricWalletRegistryEntry
...
+        const entry: FabricWalletRegistryEntry = entries.find((item: FabricWalletRegistryEntry) => {
+            if (item.fromEnvironment) {
+                return item.name === name && item.fromEnvironment === fromEnvironment;
+            } else {
+                return item.name === name;
+            }
+        });
+
+        if (!entry) {
+            throw new Error(`Entry "${name}" from environment "${fromEnvironment}" in registry "${FileConfigurations.FABRIC_WALLETS}" does not exist`);
```
Comment (Jakeeyturner) [inline packages/blockchain-common/src/registries/FabricWalletRegistry.ts]: If `fromEnvironment` is undefined, then the message seems a bit odd.

```
if(!entry && fromEnvironment){
    throw new Error(`Entry "${name}" from environment "${fromEnvironment}" in registry "${FileConfigurations.FABRIC_WALLETS}" does not exist`);
} else if(!entry && !fromEnvironment){
    throw new Error(`Entry "${name}" in registry "${FileConfigurations.FABRIC_WALLETS}" does not exist`);
} else {
    return entry;
}
```

Or something else might be better!

Comment (cazfletch) [reply packages/blockchain-common/src/registries/FabricWalletRegistry.ts]: done

File: packages/blockchain-common/test/environments/AnsibleEnvironment.test.ts [hunk 2]
```diff
@@ -12,127 +12,104 @@
...
 
-    describe('#importGateways', () => {
-        it('should create all gateways', async () => {
-            const extDir: string = vscode.workspace.getConfiguration().get(SettingConfigurations.EXTENSION_DIRECTORY);
-            const homeExtDir: string = FileSystemUtil.getDirPath(extDir);
+    describe('#getGateways', () => {
+        it('should get all gateways', async () => {
 
-            await environment.importGateways();
+            await environment.getGateways();
```
Comment (Jakeeyturner) [inline packages/blockchain-common/test/environments/AnsibleEnvironment.test.ts]: You've called this but not used it (as you call it again on line 77)

Comment (cazfletch) [reply packages/blockchain-common/test/environments/AnsibleEnvironment.test.ts]: done

File: packages/blockchain-extension/test/commands/debugCommandListCommand.test.ts [hunk 3]
```diff
@@ -145,10 +141,8 @@ describe('DebugCommandListCommand', () => {
     });
 
     it('should connect to Org1 wallet before running the submit or evaluate commands', async () => {
-        const registryEntry: FabricGatewayRegistryEntry = new FabricGatewayRegistryEntry();
-        registryEntry.name = 'Org1';
-        registryEntry.associatedWallet = 'Org1';
-        registryEntry.displayName = `${FabricRuntimeUtil.LOCAL_FABRIC} - Org1`;
+        const registryEntry: FabricGatewayRegistryEntry = await FabricGatewayRegistry.instance().get('Local Fabric - Org1');
```
Comment (Jakeeyturner) [inline packages/blockchain-extension/test/commands/debugCommandListCommand.test.ts]: Could you change ```'Local Fabric - Org1'``` to be:

`${FabricRuntimeUtil.LOCAL_FABRIC} - Org1`

**Pull request merged**